### DISCUSSION TITLE
How to enable "bounce" on momentum scrolling <body> on iOS?

### DISCUSSION THREAD

**Discussion by samwillis** [ionic-team/capacitor#4206] [Q&A]
It seems by default there is no "bounce" at the top and bottom of the momentum scroll on the `<body>`, is there a way to enable it?

As some background, my app has a large scrolling `contenteditable`. iOS Safari / wkwebkit has a bug that when you scroll an `overflow: scroll` container with a selection or cursor in it the selection or cursor fade out then jump to the final position when you stop scrolling (the cursor/selecting stays visible and moves nicely when it is the `<body>` scrolling). This is un-appealing for an app which is primarily about text editing and so I'm hoping to use the main `<body>` scrolling, but despite having momentum scrolling there is no bounce at the ends, it just stop dead.

**Comment by samwillis** [comment]
To answer my own question, it is currently disabled with no way to re-enable without a custom plugin or modifying swift code:
https://github.com/ionic-team/capacitor/issues/2334

**Comment by ankitkarna99** [comment]
I have an easier method:

Follow this tutorial:
https://www.youtube.com/shorts/TlqrSpqWTV4

The tutorial is for disabling the bounce effect. Instead, name the file EnableBounce and write the following code:

```
#import <Foundation/Foundation.h>
#import <UIKit/UIKit.h>

@implementation UIScrollView (NoBounce)
- (void)didMoveToWindow {
   [super didMoveToWindow];
   self.bounces = YES;
}
@end
```
